src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmDebuggingExtension/package.json

```diff
@@ -58,7 +58,7 @@
     "extract-zip": "^2.0.1",
     "glob": "^7.1.6",
     "mocha": "^8.1.3",
-    "node-fetch": "^3.1.1",
+    "node-fetch": "^2.6.1",
```

Comment:
3.1.1 is broken for us. We'll probably still want to upgrade, but we'll need a smarter PR which is probably best left to an area expert.